File: swagger-demo/src/main/java/com/mastertheboss/SimpleProperty.java
package com.mastertheboss;

import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class SimpleProperty {
	public SimpleProperty() {

	}
 
	private String key;
	private String value;
	public String getKey() {
		return key;
	}
	public void setKey(String key) {
		this.key = key;
	}
	public SimpleProperty(String key, String value) {
		super();
		this.key = key;
		this.value = value;
	}
	public String getValue() {
		return value;
	}
	public void setValue(String value) {
		this.value = value;
	}
}

File: javaee/helloworld/src/main/java/com/mastertheboss/jaxrs/model/SimpleProperty.kt
package com.mastertheboss.jaxrs.model

import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement
class SimpleProperty {

    var key: String? = null
    var value: String? = null

    constructor() {

    }

    constructor(key: String, value: String) : super() {
        this.key = key
        this.value = value
    }
}
